ruby-challenges.rb

```diff
@@ -18,15 +38,75 @@
 # Expected output: ['Arizona', 'California', 'Idaho', 'Maine', 'Nevada', 'New Hampshire', 'Oregon', 'Rhode Island', 'Washington'] 
 
 
+
+
+
+# def hasher_to_array (hash)
+# hash.values.flatten.sort
+  
+#     end
+# p hasher_to_array(us_states)
+
+
+
+
 # --------------------3a) Create a class called Bike that is initialized with a model, wheels, and current_speed. The default number of wheels is 2. The current_speed should start at 0. Create a bike_info method that returns a sentence with all the data from the bike object.
 
+# class Bike
+#     attr_accessor :model, :wheels, :current_speed ,:faster
+
+# def initialize (model,wheels,current_speed,faster)
+# @model = model
+# @wheels = wheels
+# @current_speed = current_speed
+# @faster = faster
+
+# end
+# def get_bike_info
+#     "My fantastic bike has #{@wheels}  and that makes it easier to get to my cruising speed of #{@current_speed}, i think i might keep this new type of bike, if you want one just google  #{@model}"
+
+# end
+# def correct_way
+#     "The #{@model} has #{@wheels} and is going #{current_speed}."
+# end
+
+
+# def pedal_faster
+# "The bike is now going #{@faster}"
+
```

Comment:
We want to include our logic in the methods we create. And if we think about a method to handle going faster and one for braking, we can think that we want to use a parameter. Based on the data we actually pass, we can manipulate our current speed. I would give this one another try!